server/config/database.js

```diff
@@ -1,46 +1,20 @@
-var r = require('rethinkdb')
-				,assert = require('assert');
-
 // RethinkDB database details.
 var dbConfig = {
   host: process.env.RDB_HOST || 'localhost',
   port: parseInt(process.env.RDB_PORT) || 28015,
-  db  : process.env.RDB_DB || 'nugget',
+  db: process.env.RDB_DB || 'nugget',
   tables: {
-    'users': 'id',
-    'about': 'id'
+    'users': 'id'
   }
 };
 
-//Export the Database config details
-module.exports.dbConfig = dbConfig;
-
-//Connect to database and make a 'connection' variable avaible to be used by all other queries.
-var connection = null;
-r.connect( {host: dbConfig.host, port: dbConfig.port, db: dbConfig.db}, function(err, conn) {
-    if (err) throw err;
-    connection = conn;
-});
-
-/*==================
-All Query Functions
-====================*/
-
-//Sample query to fetch about
-module.exports.getAbout = function(callback){
-	r.table('about').limit(1).run(connection, function(err, cursor) {
-    if (err) throw err;
-    
-    cursor.toArray(function(err, result) {
-        if (err) throw err;
-        callback(null, result);
-    });
-	});
-}
+// Initiate the Database connection
+var thinky = require('thinky')(dbConfig);
+var r = require('rethinkdb'), assert = require('assert');
 
-//Setup the Database and create tables, executed only when app starts.
-module.exports.setup = function() {
-  r.connect({host: dbConfig.host, port: dbConfig.port }, function (err, connection) {
+// Setup the Database
+thinky.databaseSetUp = function(){
+  r.connect({host: dbConfig.host, port: dbConfig.port}, function (err, connection) {
     assert.ok(err === null, err);
```

Comment:
What exactly is assert doing here? Is it just checking if there is an error? If that's all it's doing, an if statement makes more sense than requiring an entire library just to check.